EXT_WAVEFRONT.md

```diff
@@ -9,16 +9,44 @@ Introduction
 
 This extension is specifically designed for the domain of coherent wavefront propagation codes.
 
-Additional attributes on `series`
----------------------------------
+
+
+Mesh Based Records (Fields)
+---------------------------
+
+### Naming Conventions for `mesh record`s (field records)
+
+When added to an output, the following naming conventions shall be used for complex
+electric field `mesh records`. 
+
+
+- `electricField/` 
+  - type: *(complexX)*
+  - scope: *(required)*  
+  - components:
+    - `x/` 
+    - `y/` 
+    - `z/`     
+  - decription: electric field representing the wavefront. The field oscillate as $\exp(-i \omega t)$, where the angular frequency $\omega$ is related to `photonEnergy` defined below. The real field at a time $t$ is then $\Re($ `electricField` * $\exp(-i \omega t)$)
+  
+  - advice to implementors: if attribute `temporal domain` is `'time'`, this is an electric field with SI unit in `V/m`, and therefore must have:
+    - `unitDimension = (1, 1, -3, -1, 0, 0, 0)`  (electric field)
+  - advice to implementors: if attribute `temporal domain` is `'frequency'`, this must have:
+    - `unitDimension = (0. -1, 0, 0, 0, 0, 0)` (inverse length, as $\sqrt{\textrm{J} / \textrm{eV} }/ \textrm{m}  = \textrm{m}^{-1}$)
+
+
+### Additional attributes on `mesh record`s (field records)
+
 On the `series` object, set the following attributes:
 
-  - `beamline`
-    - type: *(string)*
-    - scope: *optional*
-    - description: The string representation of the optical beamline.
+  - `photonEnergy`
+    - type: *(floatX)* 
+    - description: The central photon energy of the wavefield $E_\textrm{photon} = \hbar \omega $, where $\hbar$ is Planck's constant divided by $2\pi$.
+    - scope: *required*
+    - `unitDimension = (2., 1., -2., 0., 0., 0., 0.)` (energy)
```

Comment:
attributes do not have attributes themselves. You could instead write here that the numerical value of the `photonEnergy` attribute shall be specified in Joules/eV/keV/MeV or so (same below).

Reply:
@ax3l Ah, right. But, do we delete this last line, or further add, 'unitSI =1', or simply say what you suggest?